java/tests/unittests/com/google/idea/blaze/java/run/producers/BlazeJUnitTestFilterFlagsTest.java

```diff
@@ -63,6 +71,25 @@ public void testParameterizedIgnoredForSingleClass() {
         .isEqualTo("com.google.idea.ClassName#");
   }
 
+  @Test
+  public void testParameterizedIgnoredForSingleClassJUnit5() {
+    assertThat(
```

Comment:
what is the difference between this test and `testSingleJUnit5ClassFilter`?

Reply:
You're absolutely right, there's no difference. That means the test above it (the same, but for junit4) is broken as well. Will fix them both and make them meaningful.